### ISSUE TITLE
Input.button with onPress = Nothing is not disabled

### ISSUE THREAD

**Issue by kuon** [mdgriffith/stylish-elephants#101]
When `Input.button` has `Nothing` for `onPress` the `div` with `role="button"` should have the attribute `aria-disabled` (https://www.w3.org/TR/wai-aria-1.1/#aria-disabled).

https://ellie-app.com/QPMN8xLxcZa1
